packages/@aws-cdk/aws-codepipeline-actions/lib/codestar-connections/source-action.ts

```diff
@@ -8,6 +8,24 @@ import { sourceArtifactBounds } from '../common';
 // eslint-disable-next-line no-duplicate-imports, import/order
 import { Construct } from '@aws-cdk/core';
```

Comment:
Can you add a section to the ReadMe of the `@aws-cdk/aws-codepipeline-actions` module, explaining how to use the Variables? Like we do for the other actions, [example](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-codepipeline-actions/README.md#aws-codecommit) (scroll down for the variables paragraph in that section).

Reply:
Done, will land in 3c29e3a0b